app/views/users/_profile.html.erb

```diff
@@ -87,6 +94,44 @@
     </div>
   <% end %>
 
+  <% users_setting = @user.setting %>
```

Comment:
I find this naming confusing, why the plural `users_`?

Reply:
That was how the table was named when I assumed responsibility for this RFC, MK. In my mind, I see it as the Setting for the Users table 🤷🏾‍♀️ 

I will say a big part of the work has been keeping track of the correct terminology, depending on context 😅

Reply:
Giving how ubiquitous this is and since things are mostly tested at this point I'm inclined to leave this as is for now and we can follow up refactor if we want to. 
<img width="381" alt="Screen Shot 2021-07-02 at 3 57 39 PM" src="https://user-images.githubusercontent.com/1813380/124322961-08659200-db46-11eb-9bb4-26df80590ecb.png">

Reply:
@msarit 

> That was how the table was named when I assumed responsibility for this RFC, MK. In my mind, I see it as the Setting for the Users table 🤷🏾‍♀️

I understand, but this variable holds the settings for one user, not all of them, that's why I found it potentially confusing. It's plural on the left and singular on the right.

Anyway, it's not a big deal and if we want to we can always change it later.

Reply:
Makes sense!